paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -29,27 +27,22 @@ data class Environment(
   val assetsDir = "$appTestDir/src/main/assets/"
 }
 
-fun androidHome() = System.getenv("ANDROID_SDK_ROOT")
-    ?: System.getenv("ANDROID_HOME")
-    ?: "${System.getProperty("user.home")}/Library/Android/sdk"
+fun paparazziResourcesDetailsFilePath(): String = System.getProperty("paparazzi.file.resources-details")
 
-fun detectEnvironment(): Environment {
+fun detectEnvironment(paparazziResourcesDetailsFile: String): Environment {
```

Comment:
Per the earlier feedback, let's remove this argument and instead do something like:

```
  val resourcesFile = File(System.getProperty("paparazzi.test.resources"))
```

Reply:
having this as an argument will make testing easier: a method with clear inputs and outputs. I suggest to bring back the default value to the argument.

Reply:
I'd prefer `detectEnvironment` to remain parameter-less and instead users wishing to customize can instead create a custom Environment.